indico/modules/designer/util.py

```diff
@@ -32,6 +32,14 @@ def get_nested_placeholder_options():
     return groups
 
 
+def get_images_placeholder_types():
+    image_types = []
+    for name, placeholder in get_placeholder_options().viewitems():
+        if getattr(placeholder, 'is_image', False):
+            image_types.append(name)
+    return image_types
```

Comment:
This can be simplified a lot:

```suggestion
    return [name for name, placeholder in get_placeholder_options().viewitems() if placeholder.is_image]
```

to avoid the getattr check you need to set `is_image = False` in the `DesignerPlaceholder` base class, but I think makes sense anyway:

```python

    #: Whether this placeholder is rendering an image
    is_image = False
```

Reply:
Yes, I agree.